Add CPU usage percentage to px

noahfx opened the pull request:
This will change:
-  CPU heading for CPU TIME
 
Will add:

-  CPU column with the CPU use percentage

This closes https://github.com/walles/px/issues/45

noahfx committed:
Add CPU usage percentage to px

noahfx committed:
Update tests for new CPU output

noahfx committed:
Fix comments to be compliant with project style

walles commented on tests/px_terminal_test.py:
```diff
@@ -10,8 +10,8 @@
-        "  PID COMMAND USERNAME   CPU RAM COMMANDLINE",
-        "47536 fluff   root     0.03s  0% /usr/bin/fluff 1234"
+        "  PID COMMAND USERNAME CPU CPU TIME RAM COMMANDLINE",
+        "47536 fluff   root      0% 0.03s  0% /usr/bin/fluff 1234"
```
The spacing in this second line is mis-aligned with the heading. `/usr/...` should start right below `COMMANDLINE`. And having the CPU Time column being right-aligned would help readability IMO.

walles committed:
Call CPU% "cpu_percent"

This is more precise than "cpu_usage".

walles committed:
Make headings a somewhat more obvious

All previous headings were single word only, which made it obvious where
the boundaries between the headings were.

This change makes the actual heading a bit uglier IMO, but it still
improves on the readability of the table.

walles commented:
@noahfx I like it! A colleague of mine said he was tricked by the multiple `%` columns, so I think this is a great change.

I wanted to modify the patch a bit but was unable to push to your branch, so I did them in #47 instead.

Can you review there please?

walles commented:
Or you could do this:
```sh
git fetch --all
git cherry-pick 3757491d34ba564c66831e4459a360daf8990a3b
git cherry-pick 893b5715af47a309dc38687fac8a376a6dd5e235
```

Assuming you approve of my changes of course.

That's probably better, this way Github will credit you for the PR rather than me.

noahfx commented:
Thanks! 
Those changes are OK with me, I have applied them to my PR branch 

P.S: Yesterday I uploaded the px deb package to the Debian Archive, it stills in the NEW Queue since its a new package it needs to be reviewed by the FTP Masters. https://ftp-master.debian.org/new/px_1.0.12-1.html

walles commented:
Merged, thanks for your patch and for the Debian packaging status update :)

walles commented:
@noahfx, I've been considering when to make another release with this change, but do you think I should wait until the Debian package has been approved by the FTP Masters?

noahfx commented:
@walles Feel free to make the release, as soon as the package is approved I will upload the newer version
